mono/mini/mini-x86.h

```diff
@@ -315,6 +315,7 @@ typedef struct {
 	/* Only if storage == ArgValuetypeInReg */
 	ArgStorage pair_storage [2];
 	gint8 pair_regs [2];
+	gboolean pass_empty_struct; // Set in scenarios when empty structs needs to be represented as argument.
```

Comment:
We should convert all gboolean fields to use a bitfield and avoid wasting space.